openedx/core/djangoapps/plugins/constants.py

```diff
@@ -1,25 +1,16 @@
-# Name of the class attribute to put in the AppConfig class of the Plugin App.
-PLUGIN_APP_CLASS_ATTRIBUTE_NAME = u'plugin_app'
 
-
-# Name of the function that belongs in the plugin Django app's settings file.
-# The function should be defined as:
-#     def plugin_settings(settings):
-#         # enter code that should be injected into the given settings module.
-PLUGIN_APP_SETTINGS_FUNC_NAME = u'plugin_settings'
-
-
-class ProjectType(object):
+class ProjectType():
     """
     The ProjectType enum defines the possible values for the Django Projects
     that are available in the edx-platform. Plugin apps use these values to
     declare explicitly which projects they are extending.
     """
-    LMS = u'lms.djangoapp'
-    CMS = u'cms.djangoapp'
 
+    LMS = "lms.djangoapp"
```

Comment:
Nit: I like single quotes. 🙂

Reply:
Not sure why these were changed, I think black might have done this, though I will have to check.